containers/Makefile

```diff
@@ -149,6 +152,9 @@ build/concent-api/requirements.lock: build/concent-api/concent-api.tar
 		--directory build/concent-api/           \
 		concent_api/requirements.lock
 
+build/%/cacheable-requirements.lock: build/%/requirements.lock
+	awk '$$2!="../middleman_protocol"' "$<" > "$@"
```

Comment:
Why `%` in the rule? This rule is specific to middleman. We don't want it to kick in if we add `cacheable-requirements.lock` elsewhere in the future.

Reply:
This rule is uses as dependency in `concent-api` and `signing-service`. The `%` is uses to not create separate rules. We could change name of this requirement or hard code two the same rules with different path as you want ?
I think that it's not necessary. If we want to go this direction, we must investigate all `%` in our `Makefile`.

Reply:
Right, so it's used in two places. Still, it should be more limited.

There's no problem with other `%` in the `Makefile`. They are meant to be generic. But this one is not it should work only for `concent-api` and `signing-service`. But you don't have to duplicate the rule to achieve that. You can use a [Static Pattern Rule](https://www.gnu.org/software/make/manual/make.html#Static-Pattern):

``` makefile
CACHEABLE_REQUIREMENTS_FILES :=                   \
    build/concent-api/cacheable-requirements.lock \
    build/signing-service/cacheable-requirements.lock

$(CACHEABLE_REQUIREMENTS_FILES): build/%/cacheable-requirements.lock: build/%/requirements.lock
	awk '$$2!="../middleman_protocol"' "$<" > "$@"
```

You have to repeat target names but not the commands.